index.html

```diff
@@ -78,81 +150,158 @@ <h1 class="main-text-by-form">
             </form>
           </div>
-
-          <div class="app-market">
+          <div class="top-section__app-market">
             <a href="#">
-              <img src="./img/GooglePlay.png" alt="Google Play" />
-            </a>
+              <svg class="google-play" xmlns="http://www.w3.org/2000/svg">
+                <use href="#google_market" />
+              </svg>
             <a href="#">
-              <img src="./img/AppStore.png" alt="App Store" />
+              <svg class="app-store">
+                <use href="#app-store" />
+               </svg>
             </a>
           </div>
         </div>
       </div>
     </main>
-    <div>
-      <h1 class="header">What do we offer</h1>
-      <div class="line">
-        <hr class="line" />
-      </div>
-      <div class="container-offer">
-        <div class="content">
-          <img class="support-img" src="./img/support_icon.png" alt="support" />
-          <p class="text">Support 24/7</p>
-        </div>
-        <div class="content">
-          <img
-            class="communicate-img"
-            src="./img/communicate_icon.png"
-            alt="communicate"
-          />
-          <p class="text">Communicate directly</p>
-        </div>
-        <div class="content">
-          <img class="book-img" src="./img/book_icon.png" alt="book" />
-          <p class="text">Book online</p>
-        </div>
-        <div class="content vektor-review">
-          <img class="review-img" src="./img/reviews_icon.png" alt="reiews" />
-          <p class="text">Real guest reviews</p>
+    <section>
+      <div class="wrapper">
+        <div class="offer-section">
+          <h1 class="offer-section__header">What do we offer</h1>
+          <div class="offer-section__container">
+            <div class="offer-section__container--content col-lg-3 col-md-6 col-sm-6 " >
+              <svg class="support-icon">
+                <use href="#support-icon" />
+               </svg>
+              <p class="text-icon">Support 24/7</p>
+            </div>
+            <div class="offer-section__container--content col-lg-3 col-md-6 col-sm-6 ">
+              <svg class="communicate-icon">
+                <use href="#communicate-icon" />
+               </svg>
+
+              <p class="text-icon">Communicate directly</p>
+            </div>
+            <div class="offer-section__container--content col-lg-3 col-md-6 col-sm-6  book">
+              <svg class="book-icon">
+                <use href="#book-icon" />
+               </svg>
+
+              <p class="text-icon">Book online</p>
+            </div>
+            <div class="offer-section__container--content col-lg-3">
+              <svg class="reviews-icon">
+                <use href="#reviews-icon" />
+               </svg>
+
+              <p class="text-icon">Real guest reviews</p>
+            </div>
+          </div>
         </div>
       </div>
-    </div>
+    </section>
```

Comment:
![image](https://user-images.githubusercontent.com/79144475/203797534-961dc6b2-6d72-43a9-96a3-f8242843857f.png)
check adaptive styles, in some resolutions - broken

Reply:
![image](https://user-images.githubusercontent.com/79144475/203797895-fbbdcde9-5a08-42b0-8c60-73b6ae4badd8.png)

Reply:
and check width of page in adaptive version